ci-operator/config/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-master.yaml

```diff
@@ -141,7 +141,7 @@ tests:
   optional: true
   steps:
     cluster_profile: packet-assisted
-    workflow: baremetalds-assisted
+    workflow: assisted-baremetal
```

Comment:
all ``*priv*`` files should be changed, and are changed automatically by the infra

Reply:
can you please elaborate?

Reply:
sure, taken from here: https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization
```
Automated tools manage all CI configuration for repositories in openshift-priv organization. Humans should not change any CI configuration related to these repositories. All manual changes to this configuration will be overwritten.
```

Reply:
reverted

Reply:
please note that after reverting those changes the ci-operator-config failed.

Reply:
I guess we have no other option in this case, because we're deleting the previous workflow
All good, I think instructions just trying to tell you: "don't expect to be able to do specific configurations in priv org, they will be overridden anyway"